### ISSUE TITLE
{'train/backbone_loss': nan, 'epoch': 0.0}

### ISSUE THREAD

**Issue by karayakar** [thomasgauthier/csm-hf#3]
Is this normal ?

![Image](https://github.com/user-attachments/assets/8483b763-7899-4ac6-b662-889d9c41cbf8)

**Comment by thomasgauthier** [comment]
No definitely not normal! Usually means something is wrong with the data or hyperparameters

**Comment by liyawei87** [comment]
can you share some training loss or samples by this train code
